code/game/objects/items/holy_weapons.dm

```diff
@@ -285,6 +285,7 @@
 /obj/item/nullrod/staff
 	icon_state = "godstaff-red"
 	inhand_icon_state = "godstaff-red"
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
handled by build_worn_icon() already

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.